persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseImporter.kt

```diff
@@ -47,7 +47,7 @@ class DatabaseImporterImpl(private val sqlDriver: SqlDriver) : DatabaseImporter
         restoreTable("User")
         restoreConversations()
         restoreTable("Message")
-        if (!fromOtherClient) restoreTable("Call")
+        if (!fromOtherClient) restoreCalls()
```

Comment:
I think we can remove restoring this table, missed calls are saved in `message` table

Reply:
Missed Calls yes, but in the future based on design we will have a new tab beside `Conversations` named `Calls` to show the history of calls, thus the main reason we have a table for storing calls.

Reply:
makes sense